src/components/forms/Checkbox.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import CheckEmptyIcon from '../../../lib/icons/react/CheckEmptyIcon';
+import CheckFilledIcon from '../../../lib/icons/react/CheckFilledIcon';
+import CheckIndeterminateIcon from '../../../lib/icons/react/CheckIndeterminateIcon';
+
+/**
+ * @param {Object} props react props
+ * @returns {ReactElement}
+ */
```

Comment:
Are you manually typing this, or is this generated? I sort of wonder the degree of helpfulness of this... but if it is helpful, maybe we can make some standards around it?

Reply:
It's just something that gets carried over when I start a new component. There's a low chance it will ever be useful, but lots of tools play well with JSDoc so I figure it can't hurt.

As for standards, this is just optional JSDoc.